setup.py

```diff
@@ -95,14 +128,15 @@ def build_extensions(self):
         opts = self.c_opts.get(ct, [])
         link_opts = self.l_opts.get(ct, [])
         if ct == 'unix':
-            opts.append(cpp_flag(self.compiler))
-            if has_flag(self.compiler, '-fvisibility=hidden'):
-                opts.append('-fvisibility=hidden')
+            if sys.platform != 'darwin':
+                opts.append(cpp_flag(self.compiler))
+            #if has_flag(self.compiler, '-fvisibility=hidden'):
+            #    opts.append('-fvisibility=hidden')
```

Comment:
Commented out during debugging when I had missing-symbol errors, can probably be re-enabled.

Reply:
Even with this commented out, trying to use this package now *again* yields this:

```
$ python3 ./samples/simple.py
Traceback (most recent call last):
  File "pyvirtualcam/./samples/simple.py", line 4, in <module>
    import pyvirtualcam
  File "pyvirtualcam/pyvirtualcam/__init__.py", line 3, in <module>
    from .camera import Camera
  File "pyvirtualcam/pyvirtualcam/camera.py", line 13, in <module>
    from pyvirtualcam import _native_macos as _native
ImportError: dlopen(pyvirtualcam/pyvirtualcam/_native_macos.cpython-39-darwin.so, 2): Symbol not found: _OBJC_CLASS_$_NSMachBootstrapServer
  Referenced from: pyvirtualcam/pyvirtualcam/_native_macos.cpython-39-darwin.so
  Expected in: flat namespace
 in pyvirtualcam/pyvirtualcam/_native_macos.cpython-39-darwin.so
```

Not sure how I fixed it earlier.

Reply:
Fixed in https://github.com/letmaik/pyvirtualcam/pull/22.